src/Sulu/Bundle/RouteBundle/Tests/Unit/Generator/RouteGeneratorTest.php

```diff
@@ -46,8 +47,8 @@ public function testGenerate()
     {
         $entity = $this->prophesize(RoutableInterface::class);
 
-        $this->tokenProvider->provide($entity->reveal(), 'object.getTitle()')->willReturn('Test Title');
```

Comment:
think we can use `['route_schema' => '/prefix/{object.getTitle()}/postfix/{object.getId()}']` instead of `Argument::any()`?